### ISSUE TITLE
Pretrained Model Weird Output

### ISSUE THREAD

**Issue by JohnBurden** [google-research/text-to-text-transfer-transformer#439]
I am trying to do some work evaluating the capabilities of T5 and other language models. 
I have followed the notebook for deployment ( https://github.com/google-research/text-to-text-transfer-transformer/blob/master/notebooks/t5-deploy.ipynb) and this works fine.

However when I change the model to the ordinary pre-trained one (rather than the fine-tune on NQ) found on the front page (gs://t5-data/pretrained_models/3B) and follow the same steps, the output I get is mostly the double-question mark in utf-8 (0xE2 0x81 0x87 (e28187)).
This is repeated for pretty much the whole answer. Occasionaly a word from the question is in too. 

I'm not really sure what cuases this, especially when the fine-tuned model works. 

Any help would be greatly appreciated.

**Comment by craffel** [comment]
Hi, the double question mark is what the tokenizer outputs when it decodes a mask token. The pre-training objective uses these mask tokens. Without further fine-tuning and without a task prefix, it will output lots of mask tokens.

**Comment by JohnBurden** [comment]
Thanks for the information.
Is there possibly anywhere to find information about how to use task prefixes for a non-finetuned model? Thanks again.

**Comment by craffel** [comment]
See the appendix in the paper, all the prefixes are listed there.
